Makefile

```diff
@@ -29,3 +29,16 @@ test-notebook-parameters:
 	@echo "Running notebook parameters validation..."
 	pytest tests/test_notebook_parameters.py -v
 	@echo "Notebook parameters test passed :)"
+
+test-notebook-execution:
+	@echo "Running notebook execution tests..."
+	pytest tests/test_notebook_execution.py -v
+	@echo "Notebook execution tests passed :)"
+
+test-notebooks: format-notebooks-check test-notebook-parameters test-notebook-execution
+	@echo "All notebook validations completed successfully (formatting, parameters, execution) :)"
+
+test-all:
```

Comment:
To me `test-all` implies that the format checks would be run along with all of the pytest tests. Can we include those are targets that are run with this one?